python/private/coverage_deps.bzl

```
@@ -99,44 +99,28 @@ _coverage_deps = {
 
 _coverage_patch = Label("//python/private:coverage.patch")
 
-def coverage_dep(name, python_version, platform, visibility, install = True):
+def coverage_dep(name, python_version, platform, visibility):
     """Register a singe coverage dependency based on the python version and platform.
 
     Args:
         name: The name of the registered repository.
         python_version: The full python version.
         platform: The platform, which can be found in //python:versions.bzl PLATFORMS dict.
         visibility: The visibility of the coverage tool.
-        install: should we install the dependency with a given name or generate the label
-            of the bzlmod dependency fallback, which is hard-coded in MODULE.bazel?
-
-    Returns:
-        The label of the coverage tool if the platform is supported, otherwise - None.
     """
     if "windows" in platform:
         # NOTE @aignas 2023-01-19: currently we do not support windows as the
         # upstream coverage wrapper is written in shell. Do not log any warning
         # for now as it is not actionable.
-        return None
+        return
 
     python_short_version = python_version.rpartition(".")[0]
     abi = python_short_version.replace("3.", "cp3")
     url, sha256 = _coverage_deps.get(abi, {}).get(platform, (None, ""))
 
-    if url == None:
-        # Some wheels are not present for some builds, so let's silently ignore those.
-        return None
-
-    if not install:
-        # FIXME @aignas 2023-01-19: right now we use globally installed coverage
-        # which has visibility set to public, but is hidden due to repo remapping.
-        #
-        # The name of the toolchain is not known when registering the coverage tooling,
-        # so we use this as a workaround for now.
-        return Label("@pypi__coverage_{abi}_{platform}//:coverage".format(
-            abi = abi,
-            platform = platform,
-        ))
+    # Some wheels are not present for some builds, so let's silently ignore those.
+    if url != None:
```

```suggestion
    if url == None:
```

fixes the failure for me.